fetch.bs

```diff
@@ -4503,12 +4503,13 @@ steps:
     <a for="fetch params">canceled</a>:
 
     <ol>
-     <li><p>Let <var>blob</var> be <var>request</var>'s <a for=request>current URL</a>'s
-     <a for=url>blob URL entry</a>'s <a for="blob URL entry">object</a>.
+     <li><p>Let <var>blobURLEntry</var> be the <a for=url>blob URL entry</a> for the
+     <var>request</var>'s <a for=request>current URL</a>.
 
      <li>
-      <p>If <var>request</var>'s <a for=request>method</a> is not `<code>GET</code>` or
-      <var>blob</var> is not a {{Blob}} object, then return a <a>network error</a>. [[!FILEAPI]]
+      <p>If <var>request</var>'s <a for=request>method</a> is not `<code>GET</code>`,
+      <var>blobURLEntry</var> is null, or <var>blobURLEntry</var>'s object is not
```

Comment:
object should be `<a for="blob URL entry">object</a>` here, as it was before.

Reply:
Good catch!